timesheet_activity_report/report/timesheet_activity_report.py

```diff
@@ -16,106 +16,168 @@ class TimesheetReport(models.Model):
     # Field used for the Name
     _rec_name = 'activity_name'
 
-    id = fields.Integer('Id', readonly=True)
+    id = fields.Integer(
+        'Id',
+        readonly=True
+    )
     activity_type = fields.Selection(
-        [
-            ('task', 'Task'),
-            ('issue', 'Issue'),
-            ('timesheet', 'Timesheet'),
-        ], 'Type', readonly=True,
-        help="Type is based on the origin of the input (Task, Issue or "
-             "Timesheet Activities)")
-    description = fields.Char('Description', readonly=True)
+        [('task', 'Task'),
+         ('issue', 'Issue'),
+         ('timesheet', 'Timesheet'),
+         ],
+        'Type',
+        readonly=True,
+        help="""Type is based on the origin of the input (Task, Issue or
+             Timesheet Activities)"""
+    )
+    description = fields.Char(
+        'Description',
+        readonly=True
+    )
     hours = fields.Float(
-        'Time spent', digits=(16, 2), readonly=True,
-        help="Time spent on timesheet")
-    user_id = fields.Many2one('res.users', 'User', readonly=True)
-    product_id = fields.Many2one('product.product', 'Product', readonly=True)
-    date = fields.Date('Date', readonly=True)
-    project_id = fields.Many2one('project.project', 'Project', readonly=True)
-    project_state = fields.Char('State', readonly=True, help="Project State")
+        'Time spent',
+        digits=(16, 2),
+        readonly=True,
+        help="Time spent on timesheet"
+    )
+    user_id = fields.Many2one(
+        'res.users',
+        'User',
+        readonly=True
+    )
+    product_id = fields.Many2one(
+        'product.product',
+        'Product',
+        readonly=True
+    )
+    date = fields.Date(
+        'Date',
+        readonly=True
+    )
+    project_id = fields.Many2one(
+        'project.project',
+        'Project',
+        readonly=True
+    )
+#    project_state = fields.Char(

```

Comment:
@nikul-serpentcs remove dead code

Reply:
@victormartinelicocorp Removed dead code, Please review it.